README.rst

```diff
@@ -141,3 +141,56 @@ users
 There are lots of optional things. For example, look at the code in ``project/db/postgresql.conf``
 to see all the postgres tuning parameters that can be overridden by setting variables
 in pillar.
+
+
+New Relic
+---------
+
+To enable New Relic monitoring for an environment:
+
+* Get a license key for each environment where you want to use New Relic.
+* In pillar, add a variable ``secrets.NEW_RELIC_LICENSE_KEY`` containing the license key
+  for each environment::
+
+    # <environment>.sls
+    secrets:
+        NEW_RELIC_LICENSE_KEY: larrymoecurley
```

Comment:
This will be GPG-encrypted, right? Maybe show the `----BEGIN PGP MESSAGE--`, etc, to make that clear.